controllers/dscinitialization/servicemesh_setup.go

```diff
@@ -90,18 +88,60 @@ func configureServiceMeshFeatures() feature.DefinedFeatures {
 			metricsCollection, errMetrics := feature.CreateFeature("mesh-metrics-collection").
 				With(initializer.DSCInitializationSpec).
 				From(initializer.Source).
-				Manifests(
-					path.Join(templatesDir, "metrics-collection"),
-				).
 				PreConditions(
 					servicemesh.EnsureServiceMeshInstalled,
 				).
+				Manifests(
+					path.Join(feature.ServiceMeshDir, "metrics-collection"),
+				).
 				Load()
 			if errMetrics != nil {
 				return errMetrics
 			}
 			initializer.Features = append(initializer.Features, metricsCollection)
 		}
+
+		cfMaps, cfgMapErr := feature.CreateFeature("shared-config-maps").
+			With(initializer.DSCInitializationSpec).
+			From(initializer.Source).
+			WithResources(servicemesh.ConfigMaps).
+			Load()
+		if cfgMapErr != nil {
+			return cfgMapErr
+		}
+		initializer.Features = append(initializer.Features, cfMaps)
+
+		extAuthz, extAuthzErr := feature.CreateFeature("service-mesh-control-plane-setup-external-authorization").
+			With(initializer.DSCInitializationSpec).
+			From(initializer.Source).
+			Manifests(
+				path.Join(feature.AuthDir, "auth-smm.tmpl"),
+				path.Join(feature.AuthDir, "base"),
+				path.Join(feature.AuthDir, "mesh-authz-ext-provider.patch.tmpl"),
+			).
+			WithData(servicemesh.ClusterDetails).
+			PreConditions(
+				feature.EnsureCRDIsInstalled("authconfiginitializer.authorino.kuadrant.io"),
```

Comment:
```suggestion
				feature.EnsureCRDIsInstalled("authconfigs.authorino.kuadrant.io"),
```

Was this intended? Assuming it was a typo/merge mistake since I don't believe authconfiginitializer  exists as a CRD.

Reply:
Nopez, it's from renaming `Initializer` at some point.. in GoLand presumably as it does search text occurrences too. Guilty as charged. Thanks!